timemachine/md/minimizer.py

```diff
@@ -395,22 +389,23 @@ def equilibrate_host(
     group_indices = get_group_indices(bond_list, len(combined_masses))
 
     barostat_interval = 5
-    u_impls = bind_potentials(parameterize_system(hgt, ff, 0.0))  # lambda=0
+    _, params = parameterize_system(hgt, ff, 0.0)  # lambda=0
+    u_impl.set_params(flatten_params(params))
     barostat = MonteCarloBarostat(
         x0.shape[0],
         pressure,
         temperature,
         group_indices,
         barostat_interval,
         seed,
-    ).impl(u_impls)
+    ).impl([u_impl])
 
     # context components: positions, velocities, box, integrator, energy fxns
-    ctxt = custom_ops.Context(x0, v0, host_config.box, integrator, u_impls, barostat)
+    ctxt = custom_ops.Context(x0, v0, host_config.box, integrator, [u_impl], barostat)
 
-    ctxt.multiple_steps(n_steps)
+    xs, boxes = ctxt.multiple_steps(n_steps)
 
-    return ctxt.get_x_t(), ctxt.get_box()
+    return xs[-1], boxes[-1]
```

Comment:
nit, memory: can adjust `store_x_interval` so that `len(xs) == len(boxes) == 1` or so

Reply:
Are you asking for assertion? Because the default `store_x_interval` returns `len(xs) == len(boxes) == 1`

Reply:
Not entirely clear what you meant, but added assertions here: https://github.com/proteneer/timemachine/pull/1127/commits/badefc34e319083df88473e63cef240ee1af98c3

Reply:
Oh, sorry, my mistake  -- wasn't asking for an assertion, but I had misremembered and thought the default store_x_interval would result in `len(xs) == n_steps`